src/celeritas/ext/VecgeomTrackView.hh

```diff
@@ -365,12 +376,19 @@ CELER_FUNCTION real_type VecgeomTrackView::find_safety(real_type max_radius)
     CELER_EXPECT(!this->is_outside());
     CELER_EXPECT(!this->is_on_boundary());
     CELER_EXPECT(max_radius > 0);
-    real_type safety = detail::BVHNavigator::ComputeSafety(
-        detail::to_vector(this->pos()), vgstate_, max_radius);
+
+    real_type safety = BVHNavigator::ComputeSafety(
+        detail::to_vector(this->pos()),
+        vgstate_
+#if VECGEOM_VERSION < 0x020000
+        ,
+        max_radius
+#endif
```

Comment:
I think we should backport this into their BVH, since it could accelerate safety calculations in field tracking, for example.

Reply:
I will do this and create a merge request in their repository.